packages/desktop/views/dashboard/wallet/views/ImplicitAccountCreationView.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+    import { implicitAccountCreationRoute, ImplicitAccountCreationRoute } from '../index'
+    import { InitView, AccountCreation, FundConfirmation, OneTimeDeposit } from '.'
+    import { Text, TextType } from 'shared/components'
+    import { localize } from '@core/i18n'
+
+    const IMPLICIT_ACCOUNT_STEPS = Object.keys(ImplicitAccountCreationRoute).slice(1)
```

Comment:
Note for the future, each time this view mounts its likely that you will have to calculate which is the init route, as now it will always mount with the latest used router in the store or the one decided in the constructor